F1WM/Controllers/NewsController.cs

```diff
@@ -49,7 +49,100 @@ public async Task<IActionResult> GetSingle(int id)
 			}
 		}
 
-		[HttpGet("important")]
+        [HttpGet("tag/{id}")]
```

Comment:
The endpoint address is quite confusing. We're not retrieving tags here, but news itself. I think "api/news?tagId={}" would be more appropriate. The same applies to getting by category.